### ISSUE TITLE
Calling tohil-registered callback with incorrect arg count results in segfault

### ISSUE THREAD

**Issue by NasaGeek** [flightaware/tohil#57]
```python
>>> import tohil
>>> tohil.register_callback("name", lambda a: print(a))
>>> tohil.call("name", "string")
string
<tohil.tclobj: ''>
>>> tohil.call("name", "string", "string")
Segmentation fault (core dumped)
# Also occurs when passing no args
```

This could fall under the umbrella of "you did a bad thing and a bad thing happened", but I wanted to note it here. There is likely some oversight in the `PythonCmd` function causing this. Raising an exception from the callback also causes a segfault.

**Comment by lehenbauer** [comment]
Nah this should work.

**Comment by lehenbauer** [comment]
Fixed by PR #60
